test/unit/helpers/xhr/v2/IDX_RESPONSE.js

```diff
@@ -2,17 +2,30 @@ const data = {
     "state": 200,
```

Comment:
nit:
Shall we name a little more explicitly rather than just `IDX_RESPONSE` ?

Reply:
This file is actually is holding the idx object which is the mock from idx-js layer.  
I can call it 
- `IDX_OBJECT ` ?
- `IDX_OBJECT_RESPONSE` ?
 if it makes it more clear

Reply:
how about IDX_JS_RESPONSE?